### ISSUE TITLE
Wagtail text colour customisation

### ISSUE THREAD

**Issue by ellemindwave** [LDMW/cms#204]
- [x] Text on pages should be colour customisable per section

**Comment by samhstn** [comment]
Following this guide: https://blndxp.wordpress.com/2016/03/04/adding-a-color-field-to-djangowagtail/

**Comment by samhstn** [comment]
@ellemindwave If it is for every text input it may take a little longer. Having it for every input may also clutter the admin view a little. Which sections (at least initially) would you like text colour customisation for?

**Comment by samhstn** [comment]
@Danwhy do you think that once #210 is merged, we should do any further work on this to make more sections of the site have customisable text before putting this to please test?

**Comment by Danwhy** [comment]
I've only added the customisable text colour for the branding section on the resources, so we'll have to add it for any other parts that need it

**Comment by samhstn** [comment]
@ellemindwave which other parts of the site would need text colour customisation?

**Comment by ellemindwave** [comment]
@shouston3 @Danwhy At the moment the branded header is the most important part that needs colour customisable text. The body text on the rest of the resources page doesn't need it at present. 

The other area that it would be good to able to customise the text colour is the header part of the homepage, as the hero image on the homepage will change (and maybe change depending on location in the future) 

Would it be feasible for the 'east' page for example to have a different colour text to the 'west' page, for example, if that was what the relevant images needed? Or would we have to make sure all the homepage images worked with either black or white (or another specific colour) ?

**Comment by samhstn** [comment]
Currently having a different text colour depending on the image shown from the users location is not possible with the admin view.

We would have to think about a good way to do that at some point in the future, probably changing how we currently are managing the different hero images.

For now just make each hero image suit the same text colour and we can discuss this feature in a separate issue
